src/app/providers.tsx

```diff
@@ -1,13 +1,18 @@
 'use client'
 
+import '@fontsource/open-sauce-sans/300.css';
+import '@fontsource/open-sauce-sans/400.css';
+import '@fontsource/open-sauce-sans/500.css';
+import '@fontsource/open-sauce-sans/600.css';
+import '@fontsource/open-sauce-sans/700.css';
```

Comment:
I'd prefer this to be in the fonts file (deleted). Otherwise it clutters the imports section.

Reply:
Good call, updated 👍